docs/semgrep-ci/sample-ci-configs.md

```diff
@@ -84,29 +85,34 @@ To add a Semgrep configuration file in your GitHub Actions pipeline:
 :::note
 If you are self-hosting your repository, you must [use a self-hosted runner](https://docs.github.com/en/actions/using-jobs/choosing-the-runner-for-a-job#choosing-self-hosted-runners).
 :::
+
 ### Sample GitHub Actions configuration file
 
 <Tabs
     defaultValue="gha-semgrep"
     values={[
     {label: 'Default', value: 'gha-semgrep'},
-    {label: 'Semgrep Supply Chain', value: 'gha-ssc'},
+    {label: 'Semgrep OSS', value: 'gha-oss'},
     ]}
 >
 
 <TabItem value='gha-semgrep'>
 
-The following configuration creates a CI job that runs a SAST and SCA scan using both Semgrep Code and Semgrep Supply Chain.
+The following configuration creates a CI job that runs scans depending on what products you have enabled in Semgrep Cloud Platform.
 
 <GhaSemgrepAppSast />
 
+You can **run specific product scans** by passing an argument, such as `--supply-chain`. View the [list of arguments](/getting-started/cli/#scan-using-specific-semgrep-products). 
+
 </TabItem>
 
-<TabItem value='gha-ssc'>
+<TabItem value='gha-oss'>
 
-The following configuration creates a CI job that runs an SCA scan using Semgrep Supply Chain.
+The following configuration creates a CI job that runs Semgrep OSS SAST scans using rules configured for your programming language.
```

Comment:
```suggestion
The following configuration creates a CI job that runs Semgrep OSS scans using rules configured for your programming language.
```

This one I'm not too picky about; it just seemed like a lot of abbreviations in a row, but feel free to leave it if you think it's helpful (if you change it, there are several other instances of this usage)